### ISSUE TITLE
Why not improve react-native-extended-stylesheet ?

### ISSUE THREAD

**Issue by sylvainbaronnet** [jwarby/react-native-extensible-stylesheet#1]
Hi, 
It's great to see some work to improve react native stylesheet but I'm honestly wondering why  starting a new module instead of improving : https://github.com/vitalets/react-native-extended-stylesheet

**Comment by jwarby** [comment]
Hi @sylvainbaronnet, the main reason for creating a new module instead was the API design of `react-native-extended-stylesheet` - I wanted to explicitly extend existing stylesheets, rather than explicitly create a global stylesheet from which all other stylesheets then implicitly inherit from.  I was actually using that module until I hit a case where I wanted to share some style definitions as well as some variables - I then took a look at the source for that module and decided I would prefer to create a much smaller and simpler module that had the behaviour & API that I desired.

I also plan on keeping this module very small and focused - there are very few features I am considering adding:
- mixing in of other styles (similar to how we can specify multiple styles in our components with `style={[styles.myFirstStyle, styles.mySecondStyle]}`, but at the stylesheet level)
- colour manipulation (most likely by just proxying off to a library such as [tinycolor2](https://www.npmjs.com/package/tinycolor2))

**Comment by sylvainbaronnet** [comment]
Thanks, I understand now the differences. It's an interesting approach. Mixing Stylesheet could be interesting too (for managing theme for example).
If you want to keep it small maybe you could keep this module only for simple Stylesheet manipulation (extend, mixing, vars, ...) and another module for "magic" features (color, percent, @media, ...) because only adding color seems a bit "random"

**Comment by jwarby** [comment]
Yeah I still haven't decided exactly what I want to do with it, colours were just on the wish list because I'm already using `tinycolor2` in my component code to colour the TaskDescription bar on Android a bit darker than my primary colour (purely because Android automatically decides the text colour, which looked rubbish when it was black!).

Thanks for the suggestions!
